src/PeriodEnd/PaymentsDue/DeployPaymentsDue/sql/ddl/PeriodEnd.Transient.PaymentsDue.DDL.views.sql

```diff
@@ -16,9 +16,9 @@ GO
 CREATE VIEW PaymentsDue.vw_ApprenticeshipEarning
 AS
    SELECT * FROM Staging.ApprenticeshipEarnings1
-	UNION
+	UNION ALL
 	SELECT * FROM Staging.ApprenticeshipEarnings2
-	UNION
+	UNION ALL
 	SELECT * FROM Staging.ApprenticeshipEarnings3	
```

Comment:
This may have functional changes. I'm not sure that the performance increase is worth the risk